torch/csrc/jit/python/script_init.cpp

```diff
@@ -490,21 +490,6 @@ static std::shared_ptr<Graph> _propagate_and_assign_input_shapes(
   return retval;
 }
 
-static std::shared_ptr<Graph> _assign_output_shapes(
```

Comment:
Only this part of removing unused code block is related to this PR. The rest of the changes are to pass clang-tidy checks.